src/libraries/Microsoft.PowerFx.Interpreter/Microsoft.PowerFx.Interpreter.csproj

```diff
@@ -7,6 +7,11 @@
     <Configurations>Debug;Release</Configurations>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Why this change? Seems unrelated.

Reply:
Right. Was missing symbols, so I fixed it at the same time

Reply:
shouldn't we always do this, even for retail?

Reply:
Core, yes

Reply:
Let me remove that from this PR and create a new one with a change in Directory.Build.Props

Reply:
[https://github.com/microsoft/Power-Fx/pull/606](https://github.com/microsoft/Power-Fx/pull/606)